apps/prairielearn/src/pages/partials/issueBadge.ejs

```diff
@@ -1,9 +1,21 @@
 <% if (count > 0) { %>
     <% if (!(typeof suppressLink != 'undefined' && suppressLink)) { %>
         <% if (typeof issueQid != 'undefined') { %>
-            <a class="badge badge-pill badge-danger" href="<%= urlPrefix %>/course_admin/issues?q=is%3Aopen+qid%3A<%= issueQid %>">
+            <a class="badge badge-pill badge-danger" aria-label="<%= count %> 
+                <% if (count === 1) { %>
+                    open issue
+                    <% } else { %> 
+                    open issues
+                <% } %>
+            " href="<%= urlPrefix %>/course_admin/issues?q=is%3Aopen+qid%3A<%= issueQid %>">
         <% } else { %>
-            <a class="badge badge-pill badge-danger" href="<%= urlPrefix %>/course_admin/issues">
+            <a class="badge badge-pill badge-danger" aria-label="<%= count %>
+                <% if (count === 1) { %>
+                    open issue
+                <% } else { %> 
+                    open issues
+            <% } %>
+            " href="<%= urlPrefix %>/course_admin/issues">
```

Comment:
Nit (same as the above, please test):

```suggestion
            <a class="badge badge-pill badge-danger" aria-label="<%= count %> open <%= count === 1 ? "issue" : "issues" %>" href="<%= urlPrefix %>/course_admin/issues">
```

Reply:
Updated locally and tested in [72dab1f](https://github.com/PrairieLearn/PrairieLearn/pull/10496/commits/72dab1ff4062f61afcc1381c7080c6ba663b9091).